### PR TITLE
[Sample] [Graphics] - Implementation of "Displaying UltraHDR (Fresco) & (Coil)"

### PR DESCRIPTION
**Pull request by madebymozart**
# Description
This CL contains sample code for Displaying UltraHDR using the [Fresco Image Library](https://github.com/facebook/fresco) & [Coil Image Library](https://github.com/coil-kt/coil). 

Fresco - Using the `ImagePipeline`
Coil - Using Coroutines & `ImageLoader`

# How Has This Been Tested?
**Test Configuration #1 - Pixel 7 Pro API 34 Beta (Android 14 Beta)**

# Demo
Screenshot_20230919_171902

### CODE REVIEW

**Review by yrezgui — CHANGES_REQUESTED** [review]
Could you use the existing Glide fragment? With a button, you can switch between libraries. It will reduce maintenance burden

**Comment by madebymozart** [comment]
This makes sense, will do so!

File: samples/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Available Samples
 
+- [](user-interface/predictiveback/src/main/java/com/example/platform/ui/predictiveback/PBHostingActivity.kt):
```
Comment (yrezgui) [inline samples/README.md:3]: Remove this line as it's not part of this PR

File: samples/README.md [hunk 2]
```diff
@@ -24,8 +26,8 @@ Shows how to create a GATT server and communicate with the GATT client
 Demonstrates how to implement data access auditing for your app to identify 
 - [Displaying UltraHDR](graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDR.kt):
 This sample demonstrates displaying an UltraHDR image.
-- [Displaying UltraHDR (Glide)](graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRUsingGlide.kt):
-This sample demonstrates using the Glide image loading library to detect the
+- [Displaying UltraHDR (3P Libraries)](graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRUsing3PLibrary.kt):
+This sample demonstrates using the various popular image loading library to
```
Comment (yrezgui) [inline samples/README.md:30]:
```suggestion
This sample demonstrates using the various popular image loading libraries to display Ultra HDR images
```

File: samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRUsing3PLibrary.kt [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+import kotlinx.coroutines.withContext
+
+@Sample(
+    name = "Displaying UltraHDR (3P Libraries)",
+    description = "This sample demonstrates using the various popular image loading library to" +
+            " detect the presence of a gainmap to enable HDR mode when displaying an UltraHDR image",
+    documentation = "https://github.com/bumptech/glide",
+    tags = ["UltraHDR"],
+)
+@RequiresApi(VERSION_CODES.UPSIDE_DOWN_CAKE)
```
Comment (yrezgui) [inline samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/DisplayingUltraHDRUsing3PLibrary.kt:58]: Isn't Ultra HDR support backported through libraries?

**Review by yrezgui — CHANGES_REQUESTED** [review]

File: samples/README.md (hunk 1 above)
Comment (madebymozart) [reply samples/README.md]: Resolved

**Review by yrezgui — APPROVED** [review]
